libs/features/personalization/personalization.js

```diff
@@ -445,6 +445,14 @@ function registerInBlockActions(command) {
         const { fragments } = config.mep.inBlock[blockName];
         command.content = getFragmentId(command.content);
         fragments[blockSelector] = command;
+        let overridesParent = document.querySelector('var.mas-overrides');
+        if (!overridesParent) {
+          overridesParent = createTag('var', { class: 'mas-overrides' });
```

Comment:
you might want to add display:none style to mas-overrides?
i see that its not visible atm but not sure we can rely on that without explicitly hiding it